README.md

```diff
@@ -22,13 +22,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: pull-request-action
-        uses: vsoch/pull-request-action@1.0.6
+        uses: vsoch/pull-request-action@master
```

Comment:
Yes, that is a way to do it. BUT.... I suspect that a lot of folks who use vsoch/pull-request-action may
be interested in using a stable version. So, this change may have taken us to the opposite extreme. ;)
How about in the README.md file, right above line 25 you add a bold comment like:
**Using @master as an example** consider [using a released tag](https://github.com/vsoch/pull-request-action/tags) for a stable versions.

Reply:
@flavio-fernandes I do have this comment, below here https://github.com/vsoch/pull-request-action/pull/67/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32. Do you just want it in bold?

Reply:
> @flavio-fernandes I do have this comment, below here https://github.com/vsoch/pull-request-action/pull/67/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32. Do you just want it in bold?

Ah, sorry for missing that. My suggestion is to move that comment it as close to the line as possible. But it is just a suggestion; I think you got the idea.

Reply:
No worries! Let's make it bold with an **Important** so it better sticks out. I appreciate the suggestion!